### ISSUE TITLE
Add information about WRF-CTSM on the Norwegian cluster

### ISSUE THREAD

**Issue by sunnivin** [MetOs-UiO/CTSM-Norway-Documentation#16]
We should rewrite the documentation made by @gunnartl and @irismuz to the `*.md`-format and include it in the CTSM-Norway-Documentation.

**Comment by sunnivin** [comment]
Waiting for @irismuz to accept my team invitation to assign her as well.

**Comment by irismuz** [comment]
Invitation accepted!

**Comment by sunnivin** [comment]
So, @irismuz and @gunnartl. I was hoping that we can reformat your word document into the `md`-format and include it in the online documentation [here](https://metos-uio.github.io/CTSM-Norway-Documentation/wrf-ctsm/).  

Let me know if you need help with this process!

**Comment by gunnartl** [comment]
Yes! Were are closing in on a working setup, but I think its a good idea to finish it up before putting it up here.
